docs/misc/math-expr.md

```diff
@@ -22,9 +22,14 @@ title: "Expressions"
   ~ under the License.
   -->
 
-
-> This feature is still experimental. It has not been optimized for performance yet, and its implementation is known to
->  have significant inefficiencies.
```

Comment:
What changes went in to bring these expressions out of an experimental feature?

Reply:
oops sorry hit comment instead of finishing up my review, still going through the rest of these doc changes

Reply:
IIRC, a couple things:

- Improved performance by adding various optimized selectors like SingleStringInputDimensionSelector, SingleLongInputCachingExpressionColumnValueSelector, SingleStringInputCachingExpressionColumnValueSelector.
- Battle tested by having expressions be the main way of handling pre- and post-aggregation projections in Druid SQL.

Reply:
Should this be noted in the 0.18 Release notes? It looks like these changes were first included in 0.17. Not sure if this is just stale docs, or something we want to call out explicitly. As a Druid user, I just assumed this was not experimental.

cc @jihoonson

Reply:
It looks worth mentioning in the release notes. Thanks.